tests/test_irbempy.py

```diff
@@ -108,6 +108,20 @@ def test_prep_irbem(self):
             numpy.testing.assert_almost_equal(expected[key],
                                               actual[key],
                                               decimal=5)
+    def test_prep_irbem_sph_coords(self):
+        """Test prep_irbem with spherical coordinate input"""
+        loci = spacepy.coordinates.Coords([[3,0,0],[2,0,0]], 'GDZ', 'sph', use_irbem=True)
```

Comment:
Can we try something with nontrivial angles here?

Reply:
Yes, I'm kind of surprised I didn't notice that was missing from the prep_irbem test.